vms/platformvm/txs/executor/staker_tx_verification.go

```diff
@@ -165,14 +162,7 @@ func verifyAddValidatorTx(
 		return nil, fmt.Errorf("%w: %w", ErrFlowCheckFailed, err)
 	}
 
-	// Make sure the tx doesn't start too far in the future. This is done last
-	// to allow the verifier visitor to explicitly check for this error.
-	maxStartTime := currentTimestamp.Add(MaxFutureStartTime)
-	if startTime.After(maxStartTime) {
-		return nil, ErrFutureStakeTime
-	}
-
-	return outs, nil
```

Comment:
encapsulated this check to reduce diffs down the line. We also spare some code duplication